src/Microsoft.AspNetCore.Razor.LanguageServer/ProjectEngineFactory_Blazor.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using Microsoft.AspNetCore.Blazor.Razor;
+using Microsoft.AspNetCore.Razor.Language;
+using Microsoft.CodeAnalysis.Razor;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer
+{
+    // This is a verbatim copy of Blazor's project engine implementation.
+
+    internal class ProjectEngineFactory_Blazor : IProjectEngineFactory
```

Comment:
Why this wierd naming instead of `BlazorProjectEngineFactory`?

Reply:
Consistency